README.md

```diff
@@ -19,8 +19,6 @@ for updates.
 ## Requirements
 
 - [Sass](https://github.com/sass/sass) 3.3+
-- [Bourbon](https://github.com/thoughtbot/bourbon) 4.0+
-- :warning: If you need **Sass 3.2 support**, you should [use Neat 1.5.1](#installing-older-versions-of-neat)
```

Comment:
Let’s leave this note about Sass 3.2 for now; outside of the scope of removing Bourbon.